packages/exchange/src/app.module.ts

```diff
@@ -56,8 +54,6 @@ const getEnvFilePath = () => {
         ProductModule,
         AccountDeployerModule,
         AccountBalanceModule,
-        DepositWatcherModule,
-        WithdrawalProcessorModule,
```

Comment:
So Deposits and Withdrawals are disabled in the exchange?
How do we then deposit/withdraw in the future?

Reply:
We have to import this new module to have erc1888 deposit/withdrawals